yb-voyager/cmd/exportData.go

```diff
@@ -396,15 +391,15 @@ func isOracleJDBCWalletLocationSet(s srcdb.Source) (bool, error) {
 
 // https://www.orafaq.com/wiki/TNS_ADMIN
 // default is $ORACLE_HOME/network/admin
-func getTNSAdmin(s srcdb.Source) (string, error) {
+func getTNSAdmin(s srcdb.Source) string {
 	if s.DBType != "oracle" {
-		return "", nil
+		return ""
```

Comment:
what will happen later on, when this is passed as empty to debezium conf?

Reply:
Hey, this is outdated. raising an error if dbtype!=oracle

Reply:
ok